apps/services/user-profile/src/app/v2/dto/user-profileDto.ts

```diff
@@ -0,0 +1,56 @@
+import { IsBoolean, IsEnum, IsOptional, IsString } from 'class-validator'
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger'
+
+import { DataStatus } from '../../user-profile/types/dataStatusTypes'
+import { DataType } from 'sequelize-typescript'
+
+export class UserProfileDto {
+  @ApiProperty()
+  @IsString()
+  readonly nationalId: string
+
+  @ApiPropertyOptional()
+  @IsOptional()
+  @IsString()
+  readonly email?: string
+
+  @ApiPropertyOptional()
+  @IsOptional()
+  @IsString()
+  readonly mobilePhoneNumber?: string
+
+  @ApiPropertyOptional()
+  @IsOptional()
+  @IsEnum(DataType.ENUM('en', 'is'))
+  readonly locale?: string
+
+  @ApiPropertyOptional()
+  @IsOptional()
+  @IsEnum(DataStatus)
+  readonly mobileStatus?: DataStatus
+
+  @ApiPropertyOptional()
+  @IsOptional()
+  @IsEnum(DataStatus)
+  readonly emailStatus?: DataStatus
```

Comment:
I think we should just remove these fields from the DTO and I would actually want to remove them from the model and DB table in a later task.